### ISSUE TITLE
Difficulty with setup

### ISSUE THREAD

**Issue by mtvillwock** [ideo/hudson#1]
While trying to boot this app up, I had several issues, including:
`UnhandledPromiseRejectionWarning: error: role "hudson" does not exist`

Could you help with this?

**Comment by maniart** [comment]
Hi Matt, Thanks for bringing this to my attention.
You're right. I have setup my Postgres database manually, and plugged in credentials in the app.

**Comment by mtvillwock** [comment]
@maniart Yeah, I was getting that error when trying to run `node /backend/server.js` locally.
I had to do `createdb hudson` in Terminal to get it working. 🤷‍♂

**Comment by maniart** [comment]
Thanks -- this project needs a lot of love, and i"m working on putting together a list of things to make it better. Appreciate your feedback!
